flatdata-rs/lib/src/storage.rs

```diff
@@ -13,8 +13,6 @@ use std::{
     sync::Arc,
 };
 
-use diff;
```

Comment:
Is diff still used? If not, we could remove it from toml?

Reply:
Still used, but directly (`dif::xyz`)